.env.example

```
@@ -120,3 +120,7 @@ STRIPE_SECRET_KEY=sk_test_your_stripe_secret_key
 STRIPE_WEBHOOK_SECRET=whsec_your_webhook_secret
 STRIPE_PRO_MONTHLY_PRICE_ID=price_your_pro_plan_price_id
 STRIPE_PRO_YEARLY_PRICE_ID=price_your_pro_plan_price_id
+
+# Invoice from information (company name and address on invoices)
+# Use \n for line breaks, e.g., "Company Name\nAddress Line 1\nCity, State ZIP"
+INVOICE_FROM=Fake Company\nUnited States
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)